src/main/java/org/jenkinsci/plugins/ewm/actions/ExwsAllocateActionImpl.java

```diff
@@ -23,12 +26,14 @@
     private final List<ExternalWorkspace> allocatedWorkspaces = new LinkedList<>();
```

Comment:
Yes, this action currently is added only per run, so the list will not have many items in it.

But, in the future, if I'll have a different mechanism for holding the reference to all allocated workspaces, then the collection will end up getting bigger. So in that case it makes sense to have a map.